notification-sender/src/main/java/se/inera/intyg/webcert/notification_sender/notifications/services/postprocessing/NotificationResultMessageCreator.java

```diff
@@ -120,9 +122,8 @@ public void addToResultMessage(NotificationResultMessage resultMessage, Certific
 
     private void addRedeliveryMessageToResultMessage(NotificationResultMessage resultMessage,
```

Comment:
This method name still refers to a redelivery message. Would be good if it is renamed as well to something like addStatusUpdateToResultMessage..